docs/interacting/bydbctl/query/measure.md

```diff
@@ -0,0 +1,221 @@
+# Query [Measures](../../../concept/data-model.md#measures)
```

Comment:
We need a page that describes how to set the topNAggregation and retrieve the result.